core/common/src/main/java/alluxio/underfs/UnderFileSystemWithLogging.java

```diff
@@ -560,6 +560,15 @@ public boolean supportsFlush() {
   }
 
   /**
+   * This is only used in the test.
+   *
+   * @return the underlying {@link UnderFileSystem}
+   */
+  public UnderFileSystem getUnderFileSystem() {
```

Comment:
I've changed it to default. I didn't know protected methods can be accessed in the same package..